src/NUnitFramework/framework/nunit.framework.csproj

```
@@ -7,6 +7,13 @@
     <IsTestProject>false</IsTestProject>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">
+    <Title>NUnit Framework (.NET Framework 4.6.2)</Title>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <Title>NUnit Framework (.NET 6.0)</Title>
+  </PropertyGroup>
```

Maybe we can use the previously set `AssemblyConfiguration` in order not to duplicate things:

```suggestion
  <PropertyGroup>
    <Title>NUnit Framework $(AssemblyConfiguration)</Title>
  </PropertyGroup>
```